packages/code-du-travail-frontend/src/common/BigLink.js

```diff
@@ -1,16 +1,23 @@
 import React from "react";
 import PropTypes from "prop-types";
+import Link from "next/link";
 import { theme } from "@cdt/ui";
-import { Link } from "../../routes";
 import { getRouteBySource, getLabelBySource } from "../sources";
 import ArticleIcon from "../icons/ArticleIcon";
 import styled from "styled-components";
 
 export function BigLink({ icon, query, data }) {
   const { slug, title, source, path = "" } = data._source;
-  const route = getRouteBySource(source);
   return (
-    <Link route={route} params={{ q: query, slug: slug }}>
+    <Link
+      href={{
+        pathname: `/${getRouteBySource(source)}`,
```

Comment:
I thing I should do a function / utility to handle the duplicate logic here and generate the correct `href` and `as` props based on simple input

Reply:
https://github.com/zeit/next.js/issues/7329